tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -92,17 +96,31 @@ export type TimeSeriesLoadables = {
 
 export type TimeSeriesLoadable = TimeSeriesLoadables[PluginType];
 
-export interface ScalarTimeSeriesData {
-  [tag: string]: ScalarTimeSeriesLoadable;
-}
+export type TimeSeriesLoadableType =
+  | ScalarTimeSeriesLoadable
+  | HistogramTimeSeriesLoadable
+  | SampledImageTimeSeriesLoadable;
 
-export interface HistogramTimeSeriesData {
-  [tag: string]: HistogramTimeSeriesLoadable;
+export function isSampledImageTimeSeriesLoadable(
```

Comment:
I don't think this is the appropriate place to put this function. (if we need it)

Reply:
I don't love this being here either but it seemed to be the convention? i.e. [tensorboard/webapp/metrics/data_source/types.ts](https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/data_source/types.ts;l=52)

We don't really have a great spot for utility functions in this part of the code. I could imagine moving it to either [tensorboard/webapp/metrics/store/metrics_reducers.ts](https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/store/metrics_reducers.ts) or [tensorboard/webapp/metrics/store/metrics_store_internal_utils.ts](https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/store/metrics_store_internal_utils.ts).
Do you have a preference?

Reply:
Do we still need this function?